packages/@vuepress/plugin-back-to-top/BackToTop.vue

```diff
@@ -59,19 +59,19 @@ export default {
 }
 </script>
 
-<style scoped>
+<style lang='stylus' scoped>
 .go-to-top {
   cursor: pointer;
   position: fixed;
   bottom: 2rem;
   right: 2.5rem;
   width: 2rem;
-  color: #72cda4;
+  color: $accentColor;
```

Comment:
Maybe apply [lighten](http://stylus-lang.com/docs/bifs.html#lightencolor-amount) here to maintain the hover effect.

Reply:
```suggestion
.go-to-top:hover {
   color: lighten($accentColor, 30%);
 }
```

Reply:
+1

Reply:
@janvennemann @ulivz fixed in a2af31f